keyboards/spaceman/2_milk/keymaps/encg/config.h

```diff
@@ -0,0 +1,2 @@
+#define UNICODE_SELECTED_MODES UC_WINC, UC_WIN, UC_MAC, UC_LNX
```

Comment:
```suggestion
#pragma once

#define UNICODE_SELECTED_MODES UC_WINC, UC_WIN, UC_MAC, UC_LNX
```

Reply:
This change was made in latest commit along with the addition of GPL license headers